tools/cdn.sh

```diff
@@ -0,0 +1,42 @@
+EXTENSION_NAME="auth0-ldap-connector-health-monitor"
+JSON_FILE="webtask.json"
+
+# Extract full version
+FULL_VERSION=$(jq -r '.version' "$JSON_FILE")
+if [ -z "$FULL_VERSION" ] || [ "$FULL_VERSION" == "null" ]; then
+  echo "Error: Could not extract version from $JSON_FILE."
+  exit 1
+fi
+
+echo "Extracted full version: $FULL_VERSION"
+
+# Extract major.minor version
+# Use shell parameter expansion to get the major.minor version
+# ${variable%pattern} removes the shortest matching suffix pattern.
+# '.*' matches the last dot and anything following it.
+MAJORMINOR_VERSION_ONLY=${FULL_VERSION%.*}
+if [ -z "$MAJORMINOR_VERSION_ONLY" ] || [ "$MAJORMINOR_VERSION_ONLY" == "null" ]; then
+  echo "Error: Could not extract major.minor version from $JSON_FILE."
+  exit 1
+fi
+
+echo "Extracted major.minor version: $MAJORMINOR_VERSION_ONLY"
+
+deploy_bundle() {
+  BUNDLE_EXISTS=$(aws s3 ls s3://assets.us.auth0.com/extensions/$EXTENSION_NAME/ | grep "$EXTENSION_NAME-$1.js")
+
+  if [ ! -z "$BUNDLE_EXISTS" ] && [ "$1" == "$CURRENT_VERSION" ]; then
+    echo "There is already a $EXTENSION_NAME-$1.js in the cdn. Skipping cdn publish…"
+  else
+    # aws s3 cp build/bundle.js s3://assets.us.auth0.com/extensions/$EXTENSION_NAME/$EXTENSION_NAME-$1.js --region us-west-1 --acl public-read
+    eccho "Running command (dry run)"
```

Comment:
```suggestion
    echo "Running command (dry run)"
```

Reply:
do we want a dry run or a real run?

Reply:
Since originally requesting review, I realized that the existing deployment tooling is definitely broken, so I'm adding that to this PR now. That's why the script has this debugging stuff. 

I'll put this PR in draft mode and re-request review once it's deploying as expected.